editor/components/post-sticky/check.js

```diff
@@ -27,14 +23,8 @@ export function PostStickyCheck( { postType, children, user } ) {
 export default compose( [
 	withSelect( ( select ) => {
 		return {
+			post: select( 'core/editor' ).getCurrentPost(),
```

Comment:
It would be better to return only:
```js
hasStickyAction: get( post, [ '_links', 'wp:action-sticky' ], false )`
```
from `withSelect` to avoid unnecessary rerenders when the current post changes after unrelated modifications.

Reply:
Good catch, fixed in 3e591c50d08c3128745c44cb7c23d8b8ac991c5f